pkg/controller/userreport/send_test.go

```diff
@@ -80,7 +80,8 @@ func TestSendWebhookRequest(t *testing.T) {
 	ctx := project.TestContext(t)
 
 	client := &http.Client{
-		Timeout: 2 * time.Second,
+		Timeout:   2 * time.Second,
```

Comment:
We should probably bump this to 5s to match other timeouts, but wanted to limit changes in this PR